Outlines/simple-tcr.md

```diff
@@ -0,0 +1,18 @@
+# Proposed title
+Simple Token-Curated Registry
+
+## Structure
+1. What is Token-Curated Registry
+2. Examples of TCRs
```

Comment:
Let's make sure this has some good understandable examples, as reading through many posts I am still struggling to understand quite why a TCR is useful.

Reply:
Right. I was thinking about starting with a simple made-up example, and then go in-depth with some existing TCR project.